pkg/kubectl/cmd/run.go

```diff
@@ -413,7 +413,9 @@ func waitForPod(podClient coreclient.PodsGetter, ns, name string, exitCondition
 		ev, err := watch.Until(0, w, func(ev watch.Event) (bool, error) {
```

Comment:
Does this call ever return a nil object without an error?  What does the caller of `WaitForPod` do with a `nil` pod?